modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -260,24 +265,27 @@ feature -- Helpers
 
 feature {NONE} -- Queries
 
-	sql_select_nodes_count: STRING = "SELECT count(*) from Nodes;"
+	sql_select_nodes_count: STRING = "SELECT count(*) FROM Nodes WHERE status != 3;"
+			-- Nodes count (Published and not Published)
```

Comment:
Add in comment
{CMS_NODE_CONSTANTS}.trashed = 3

in order to explain why this is 3